dbt/main.py

```diff
@@ -175,6 +175,7 @@ def invoke_dbt(parsed):
             profile_to_load=parsed.profile,
             args=parsed
         )
+        proj.log_warnings()
```

Comment:
we should do this check on line 235. If `--target` is used on the CLI, then this will refer to the wrong target. Probably not an issue for most, but i just got snagged locally